pkg/server/server.go

```diff
@@ -410,3 +415,11 @@ func setMasterRoleLabel(ctx context.Context, nodes v1.NodeClient) error {
 	}
 	return nil
 }
+
+func nginxImage() string {
+	nginxImage := NginxIngressImage
```

Comment:
What's up with this?  Is nginx not a manifest list?  I'd rather maintain a manifest list ourselves or work with upstream to add one then do works around like this.